.circleci/config.yml

```diff
@@ -984,9 +1047,9 @@ jobs:
         description: List of packages to test
         type: string
     machine: true
-    resource_class: swc/ax101
+    resource_class: <<parameters.resource_class>>
```

Comment:
Why is this `swc/ax101` replaced with `<<parameters.resource_class>>`? Looks like other `swc/ax101` are replaced with `qkc/ax101`.

Reply:
This should be a parameter placeholder that can be overridden, and was unnecessarily changed from `<<parameters.resource_class>>` to `qkc/ax101` when merging code from upstream.
Now, we revert the change to maintain alignment with upstream. This change makes no difference to the result because `swc/ax101` takes effect in line 1033.